bundles/org.openhab.ui.basic/src/main/java/org/openhab/ui/basic/internal/render/SwitchRenderer.java

```diff
@@ -138,7 +138,6 @@ public EList<Widget> renderWidget(Widget w, StringBuilder sb, String sitemap) th
                 }
             }
             snippet = snippet.replace("%buttons%", buttons.toString());
-            snippet = snippet.replace("%count%", Integer.toString(nbButtons));
```

Comment:
What does this change in the Switch renderer do?

Reply:
Nothing, it was not used, so I removed it. Just cleaning.